src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,149 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { TodoElement } from '../TodoElement';
+
+enum FilterType {
+  All,
+  Active,
+  Completed,
```

Comment:
add values to your enum, so you can use it in your future `FilterButton` component